### ISSUE TITLE
Chain component + system annoying to work with + cannot work with nested Transforms

### ISSUE THREAD

**Issue by kkorolyov** [kkorolyov/pancake#84]

**Comment by kkorolyov** [comment]
less relevant after #215 + #220

**Comment by kkorolyov** [comment]
Consider simply removing `ChainSystem` as no longer relevant.
Replaceable with nested transforms?

**Comment by kkorolyov** [comment]
Replaced with `Joint` in #227.
Now `Joint` is annoying to work with...
